specification/sources/chapters/extensions/mndx/mndx_force_feedback_curl.adoc

```diff
@@ -0,0 +1,182 @@
+* pname:locations is a pointer to an array of locations to apply force
+  feedback.
+* pname:locationCount the number of locations in the locations array.
+****
+
+Contains an array of slink:XrApplyForceFeedbackCurlLocationMNDX that
+contains information on locations to apply force feedback to.
+
+include::{generated}/validity/structs/XrApplyForceFeedbackCurlLocationsMNDX.txt[]
+--
+
+[open,refpage='XrApplyForceFeedbackCurlLocationMNDX',type='structs',desc='Structure representing and containing information on how to apply force feedback to a single location.',xrefs='xrApplyForceFeedbackCurlMNDX']
+--
+The slink:XrApplyForceFeedbackCurlLocationMNDX structure is defined as:
+
+include::{generated}/api/structs/XrApplyForceFeedbackCurlLocationMNDX.txt[]
+
+.Member Descriptions
+****
+* pname:location represents the location to apply force feedback to.
+* pname:value is a value from 0-1 representing the amount of force feedback
+  to apply.
+  The range of the value should represent the entire range the location is
+  capable of moving through, with 1 representing making the location incapable
+  of movement, and 0 being fully flexible.
+  For example, in the case of a finger curl, setting pname:value to 1 would
+  prevent the finger from curling at all (fully extended), and 0 would allow
+  the finger to have free range of movement, being able to curl fully.
+****
+
+[NOTE]
+.Note
+====
+pname:value is specified as a limit in a single direction.
+For example, if the value specified is 0.5, a location must: have free
+movement from the point where it would be incapable of movement if
+pname:value was 1, to 0.5 of the range the location is capable of moving.
+====
+
+include::{generated}/validity/structs/XrApplyForceFeedbackCurlLocationMNDX.txt[]
+--
+
+*New Functions*
+
+[open,refpage='xrApplyForceFeedbackCurlMNDX',desc='Applies force feedback to a set of locations',type='protos',xrefs='']
+--
+The flink:xrApplyForceFeedbackCurlMNDX function is defined as:
+
+include::{generated}/api/protos/xrApplyForceFeedbackCurlMNDX.txt[]
+
+.Parameter Descriptions
+****
+* pname:handTracker is an slink:XrHandTrackerEXT handle previously created
+  with flink:xrCreateHandTrackerEXT.
+* pname:locations is an slink:XrApplyForceFeedbackCurlLocationsMNDX
+  containing a set of locations to apply force feedback to.
+****
+
+The flink:xrApplyForceFeedbackCurlMNDX function applies force feedback to
+the set locations listed in slink:XrApplyForceFeedbackCurlLocationsMNDX.
+
+flink:xrApplyForceFeedbackCurlMNDX should: be called every time an
+application wishes to update a set of force feedback locations.
+
+A runtime may: relax force feedback when it sees appropriate when no
+application is active, for example on start-up, or if an application has
+crashed.
+However, the runtime must: keep the last set of location information the
+application has passed until the application calls the function again, or
+the application exits.
```

Comment:
I would rephrase this in terms of session state, like the other haptic feedback stuff. (and actually does it really make sense to restore the previous limits when resuming? Lots of other things need to be re-done, I'd say that if the session leaves the focused state the force feedback is cancelled.)

Probably also say what state the session must be in to call this (probably focused)